packages/create-ship-app/src/index.js

```diff
@@ -17,12 +17,11 @@ let deploymentType;
   console.clear();
   console.log(`Hey! Let’s build your ${gradient.pastel('Ship')} 🚀`);
   
-  const args = utils.getCLIArgs();
+  projectName = utils.getCLIArgs();
   
-  if (args[0] === 'init') {
+  if (projectName === 'init') {
```

Comment:
`projectName === 'init'` looks strange 🤔 

To avoid using `args[0]` we can use array destructuring
`const [firstArg] = utils.getCLIArgs();`